contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol

```diff
@@ -10,6 +10,14 @@ import { iOVM_StateManager } from "../../iOVM/execution/iOVM_StateManager.sol";
 
 /**
  * @title OVM_StateManager
+ * @dev The State Manager contract holds all storage values for contracts in the OVM. It can only be written to by the 
+ * the Execution Manager and State Transitioner. It runs on L1 during the setup and execution of a fraud proof.
+ * The same logic runs on L2, but has been implemented as a precompile in the L2 go-ethereum client 
+ * (see https://github.com/ethereum-optimism/go-ethereum/blob/master/core/vm/ovm_state_manager.go)
+ * 
+ * Compiler used: solc
+ * Runtime target: EVM
+ * Execution environment: Layer 1 and Layer 2
```

Comment:
Because this is a precompile, this contract (as in the actual bytecode) is not executed on L2 nodes, so depending on our definition of EE here, might wanna say only L1.